src/main/webapp/WEB-INF/pages/manage/overview.ftl

```diff
@@ -108,6 +108,7 @@
 
         $('.confirm').jConfirmAction({titleQuestion : "<@s.text name="basic.confirm"/>", question : "<@s.text name='basic.confirm'/>", yesAnswer : "<@s.text name='basic.yes'/>", cancelAnswer : "<@s.text name='basic.no'/>"});
         $('.confirmRegistration').jConfirmAction({titleQuestion : "<@s.text name="basic.confirm"/>", question : "<@s.text name='manage.overview.visibility.confirm.registration'/> <@s.text name='manage.resource.delete.confirm.registered'/>", yesAnswer : "<@s.text name='basic.yes'/>", cancelAnswer : "<@s.text name='basic.no'/>", checkboxText: "<@s.text name='manage.overview.visibility.confirm.agreement'/>"});
+        $('.confirmEmlReplace').jConfirmAction({question : "<@s.text name='manage.metadata.replace.confirm'/></br>", yesAnswer : "<@s.text name='basic.yes'/>", cancelAnswer : "<@s.text name='basic.no'/>"});
```

Comment:
Do we need '</br> here?
Add a titleQuestion please (otherwise it won't be translated)

Reply:
Done.
Indeed, just tested, <br/> is no more required with bootstrap version.